<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(100, 100, 100, 255));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="ColorFrame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>681</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeColor">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
   padding: 5px;
   background: #00ffff;
 border: 1px solid #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Change Color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ColorLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
padding: 5px;
background: #ddfdff;
border: 1px solid #dddddd;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>510</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
padding: 5px;
background: #ffddfd;
border: 1px solid #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>12:RGB ↔ XYZ ↔ HSL</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
    background: #a0a6ff;
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;</string>
    </property>
    <property name="title">
     <string>RGB</string>
    </property>
    <widget class="QLineEdit" name="RLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #ff898b;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="GLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #ff898b;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #ff898b;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QSlider" name="RSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="GSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="BSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
    background: #a0a6ff;
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;</string>
    </property>
    <property name="title">
     <string>XYZ</string>
    </property>
    <widget class="QLineEdit" name="XLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #ffff7f;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="YLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #ffff7f;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ZLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #ffff7f;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Xlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="YLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="ZLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QSlider" name="XSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="YSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="ZSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
    background: #a0a6ff;
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;</string>
    </property>
    <property name="title">
     <string>HLS</string>
    </property>
    <widget class="QLineEdit" name="HLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #55ff7f;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #55ff7f;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
padding: 5px;
background: #55ff7f;
border: 1px solid #dddddd;</string>
     </property>
    </widget>
    <widget class="QLabel" name="HLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
    <widget class="QLabel" name="LLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QLabel" name="SLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>L</string>
     </property>
    </widget>
    <widget class="QSlider" name="HSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="SSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="LSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="ErrorsLine">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial, sans-serif;
    font-size: 14px;
    padding: 5px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #beffa0, stop:1 #f5f5f5);
    border: 1px solid #dddddd;
    border-radius: 5px;
    color: #333333;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
